tests/test_generic/tests.py

```diff
@@ -164,9 +165,9 @@ def test_iban_fields(self):
             'MU17 BOMM 0101 1010 3030 0200 000M UR': 'MU17BOMM0101101030300200000MUR',
             'MU 17BO MM01011010 3030-02 000-00M UR': 'MU17BOMM0101101030300200000MUR',
 
-            'BE68539007547034': 'BE68539007547034',
-            'BE68 5390 0754 7034': 'BE68539007547034',
-            'BE-685390075470 34': 'BE68539007547034',
+            'BE31538007547055': 'BE31538007547055',
+            'BE31 5380 0754 7055': 'BE31538007547055',
+            'BE-315380075470 55': 'BE31538007547055',
```

Comment:
The example BE IBAN that we were using (from the SWIFT IBAN docs) passes the IBAN validation, but not the stricter validation that stdnum.iban uses for some countries. That's the reason I needed to use a different IBAN here.